kotest-tests/kotest-tests-android/android-library/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+@file:Suppress("UnstableApiUsage")
+
+plugins {
+   id("com.android.library") version "7.3.0"
+   id("org.jetbrains.kotlin.android") version "1.8.0"
+}
```

Comment:
@gabrielittner I've added a basic integration test for verifying that `@Language` on Android works, but I'm not familiar with Android development so I'm not sure if the project I've created will trigger the same problem that you had. Would you be able to take a look?

Reply:
If you (locally) revert [efb6e2a](https://github.com/kotest/kotest/pull/3397/commits/efb6e2a7b6b52b25fb40d2db9ea329e1687b97c7) and run the test, do they fail?

Reply:
Yeah I've reverted it locally so that `@ org.intellij.lang.annotations.Language` is defined in commonMain, which should reproduce the problem.

I think in order for the Android duplication check to be triggered the classes _have_ to come via a Maven dependency. When I add in 

```kotlin
implementation("io.kotest:kotest-common:5.5.5")
```

the test fails, but I've hacked it to use JARs.

I'll try and get a proper integration test working tomorrow...

Reply:
Integration test is working - I moved it to another PR #3401, because this PR is a quick fix that I'm confident will work.